### PR TITLE
Plant consistency and flavor text

### PR DESCRIPTION
**Pull request by alsoandanswer**
* Changes the plant sprites to be more consistent.

* Using the same pot might open the gates for some coder smarter than me to make plants in the pots changeable in the future.

* Adds flavortext to some plants. May or may not spark RP. I hope the Loredevs won't get mad.

forum thread here : https://forums.aurorastation.org/topic/11559-stoners-pot-palace-plants-same-pot-consistency-abound-and-flavortext-to-boot/

### IC changelog
I mean, it makes more sense that Nanotrasen would save money by bulk-buying one pot.
pot

### CODE REVIEW

File: code/game/objects/structures/flora.dm [hunk 1]
```diff
@@ -142,10 +142,116 @@
...
 	var/number = rand(1,36)
 	if (number == 36)
-		if (prob(90))//Make the wierd one rarer
+		if (prob(90))//Make the weird one rarer
 			number = rand(1,35)
 		else
-			desc = "It stares into your soul."
+			desc = "A half-sentient plant borne from a mishap in a Zeng-Hu genetics lab."
+
+	if (number == 1)
```
Comment (Sindorman) [inline code/game/objects/structures/flora.dm]: this chain of if statements is really ugly. make it into switch statements

```
switch(number)
    if(1)
          desc = "Just your common, everyday houseplant." // cool flavortext for plants
    if(2)
          desc = "Just your common, everyday houseplant."
```

And just follow that pattern.

Comment (Sindorman) [reply code/game/objects/structures/flora.dm]: Also I noticed that most of them have literally same description `desc = "Just your common, everyday houseplant."`. Please make special cases as `if` and anything else can go into `else`

Comment (Sindorman) [reply code/game/objects/structures/flora.dm]: So remove 1, 2, 6, 8, 10, 11, 12, 16, 17, 21, 23, 24, 25, 26, 28, 29, 30, 31, 35. Leave the rest into `switch(number)` statement as ifs and like things above that I mention will go into 
```
else
    desc = "Just your common, everyday houseplant."
```

Comment (alsoandanswer) [reply code/game/objects/structures/flora.dm]: Ah, thanks Sindorman. I was trying to figure that out for the longest time until I gave up and brute-forced it.

Comment (alsoandanswer) [reply code/game/objects/structures/flora.dm]: Fixed.

**Comment by alsoandanswer** [comment]
!review

**Pull request merged**